spec/rails_helper.rb

```diff
@@ -108,6 +108,8 @@ class Analytics
     descendants.each(&:disable_test_adapter)
   end
 
+  WebMock.allow_net_connect!(net_http_connect_on_start: true)
```

Comment:
This was necessary to get around a misleading "Too many open files" error.
https://github.com/bblimke/webmock/blob/master/README.md#connecting-on-nethttpstart